qiskit/test/mock/fake_backend.py

```diff
@@ -96,16 +96,13 @@ def properties(self):
 
         return BackendProperties.from_dict(properties)
 
-    def run(self, qobj):
+    def run(self, _):
```

Comment:
This is defined in the abstract class that this fake object is building, we shouldn't change the name here so this fake truly matches what the abstract class is doing.